src/frontend/src/components/GitHubContributorCard/GitHubContributorCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import { Card, CardContent, Typography, Avatar, Grid } from '@material-ui/core';
+import useTelescopeContributor from '../../hooks/use-telescope-contributor';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.primary.light,
+  },
+  typography: {
+    fontSize: 13,
+  },
+  links: {
+    color: theme.palette.text.secondary,
+    textDecorationLine: 'none',
+    '&:hover': {
+      textDecorationLine: 'underline',
+    },
+    alignItems: 'flex-start',
+  },
+}));
+
+const GitHubContributorCard = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const { contributor, error } = useTelescopeContributor();
+
+  return { contributor } ? (
+    <Card className={classes.root}>
+      <CardContent>
+        <Grid container direction="row" alignItems="center">
+          <Grid item xs={2}>
+            <Avatar alt="Profile Picture" src={contributor?.avatar_url} />
+          </Grid>
+          <Grid item xs={10} justify="flex-end">
+            <Typography className={classes.typography}>
+              Thank you{' '}
+              <a href={contributor?.html_url} className={classes.links}>
+                {contributor?.login}
+              </a>
+              , for being a{' '}
```

Comment:
&nbsp instead of {' '}.

[nbsp example](https://www.w3schools.com/html/html_entities.asp#:~:text=A%20commonly%20used%20entity%20in,the%20words%20might%20be%20disruptive.)